resources/styles/components/performance.less

```diff
@@ -0,0 +1,18 @@
+.performance-report {
+  table {
+    th {
+      .no-select();
+      &.sortable {
+        cursor: pointer;
+      }
+      &.is-ascending:after {
+        content: '\00a0\f0de';
```

Comment:
I believe there is a `.fa-*()` mixin that adds this char as well as the line below this.